src/App.tsx

```diff
@@ -1,61 +1,101 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
```

Comment:
```suggestion
  const [hasTitleError, setHasTitleError] = useState(false);
```
Boolean variables should start from `is` or `has`